orchestra/tests/helpers/fixtures.py

```diff
@@ -39,49 +39,54 @@
 ITERATION_DURATION = timedelta(hours=1)
 PICKUP_DELAY = timedelta(hours=1)
 
-TODO_TEMPLATE_GOOD_CSV_TEXT = """Remove if,Skip if
-[],[],the root
-[],[],,todo parent 1
-[],"[{""prop"": {""value"": true, ""operator"": ""==""}}]",,,todo child 1-1
-"[{""prop"": {""value"": true, ""operator"": ""==""}}]",[],,todo parent 2
-[],[],,,todo child 2-1
-[],[],,,,todo child 2-1-1
-[],[],,,todo child 2-2
+TODO_TEMPLATE_GOOD_CSV_TEXT = """Slug,Remove if,Skip if
```

Comment:
Can you test the import and export of one of these rows with a slug, while you leave the rest empty?